packages/element-library/src/gif/test/output.js

```diff
@@ -109,6 +109,8 @@ describe('Gif Output', () => {
   });
 
   describe('AMP validation', () => {
+    jest.retryTimes(3, { logErrorsBeforeRetry: true });
```

Comment:
So it only affects the AMP validation related tests.

Reply:
To further clarify: as per the [documentation](https://jestjs.io/docs/jest-object#jestretrytimesnumretries-options), `jest.retryTimes` can live either at the top-level of a test file or in a `describe` block.

Here, I chosen to add it only within the `describe` block related to AMP validation because those tend to be the flaky tests (because of the AMP validator WASM). The other unit tests are not usually flaky at all, so it would be a bit wasteful to retry those automatically.